CONTRIBUTING.md

```diff
@@ -57,4 +57,30 @@ Feel free to explore beauty of Python programming. Make contributions that matte
 
 The answer to the question "When is it ready?" is "When it is ready" :-)
 
+## Django's template standard
 
+There are few simple rules:
+
+1. Blank line between ``load`` and rest of template.
+2. Blank line between block section.
+3. If you open some HTML tag or Django template tag you must indent line, exclude simple code (see example).
+
+All of this is for human readability not for browsers.
```

Comment:
We prefer keeping *source* template clean, not bothering with the output html indentation, which could be unreadable.